server/src/main/java/com/google/sps/utils/AdRowProcessor.java

```diff
@@ -0,0 +1,129 @@
+/* 
+ * Description: File for building ads and adding them to Firestore
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */ 
+package com.google.sps.utils;
+
+import com.google.api.core.ApiFuture;
+import com.google.auth.oauth2.GoogleCredentials;
+import com.google.cloud.firestore.DocumentReference;
+import com.google.cloud.firestore.Firestore;
+import com.google.cloud.firestore.WriteResult;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.FirebaseOptions;
+import com.google.firebase.cloud.FirestoreClient;
+import com.google.sps.utils.Ad.Builder;
+import com.google.sps.utils.Ad;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class AdRowProcessor {
+
+  private static final String PATH_TO_SERVICE_ACCOUNT = "./serviceAccountKey.json"; 
+  private static final String DATABASE_URL = "https://step9-2020-capstone.firebaseio.com"; 
+  private String[] row; 
+
+  public AdRowProcessor(String[] csvRow) throws Exception {
+    this.row = csvRow; 
+    // initialize app
```

Comment:
In general, when using comments to separate sections, add a new line before the comment.

Also capitalize the first letter of the comment and put a period at the end.

Also elsewhere.